lib/gems/pending/appliance_console/cli.rb

```diff
@@ -286,6 +293,15 @@ def extauth_opts
       extauthopts.update_configuration(extauthopts_hash)
     end
 
+    def server_mode
+      service = LinuxAdmin::Service.new("evmserverd")
+      case server_state
+      when :status then service.running? ? "running" : "not running"
```

Comment:
I think this method could be simplified if you checked `service.running?` outside of the case statement.